packages/std/src/testing/mock.rs

```diff
@@ -484,6 +490,33 @@ impl<C: DeserializeOwned> MockQuerier<C> {
         self.bank.set_denom_metadata(denom_metadata);
     }
 
+    #[cfg(feature = "cosmwasm_1_3")]
+    pub fn set_withdraw_address(
+        &mut self,
+        delegator_address: impl Into<String>,
+        withdraw_address: impl Into<String>,
+    ) {
+        self.distribution
+            .set_withdraw_address(delegator_address, withdraw_address);
+    }
```

Comment:
I added the requested methods now.
One thing I'm not quite sure about is all the forwarding. It's necessary because we don't expose the `MockQuerier`'s `DistributionQuerier`. Maybe it would make more sense to expose it instead?
@webmaster128 Would love to hear your opinion on this

Reply:
You are right, this is very questionable. I'd say historical reasons. Could you open a ticket to refactor this? But let's go with the flow now and get 1.3 out.